src/main/java/com/amazon/opendistroforelasticsearch/security/user/User.java

```diff
@@ -265,6 +265,9 @@ public final void addOpenDistroSecurityRoles(final Collection<String> securityRo
     }
 
     public final String getUserRolesString() {
-        return name + "|" + String.join(",", getRoles()) + "|" + String.join(",", getOpenDistroSecurityRoles());
+        return name + "|"
```

Comment:
thanks @akbhatta . Could you pls add an overloaded method instead of changing existing method to main backward compatibility ? something like  `getUserRolesTenantString`

Reply:
Then I guess, you need to change thread_context string also. Probably, its fine then, make sure that you change common_utils also.

Reply:
Good point. I thought about it. I am planning to put overloaded methods to common-utils:User.java
Here, I don't see any usage of this function except while adding to thread context. may be renaming the method to  getUserRolesTenantString would suffice?

Reply:
Yes. I will post a PR for common-utils soon. I like the idea of renaming  getUserRolesString --> getUserRolesTenantString
OR getUserRolesString --> getThreadContextTransientString. Let me know your preference.

Reply:
If you overload, you need to add one more transport thread_context string , so leave it like that. And change common-utils. Sounds good.

Reply:
will vote for: getUserRolesTenantString

Reply:
Agreed in meeting to change it to getThreadContextTransientString and rename the value as well to make name agnostic in the future.